### PR TITLE
Added documentation for stock ownership

### LINKED ISSUE #47: Add documentation for stock ownership
(Once/if https://github.com/inventree/InvenTree/pull/1155 is approved, also relates to https://github.com/inventree/InvenTree/issues/1129)

Explain:
* How to enable stock ownership feature
* How to set owner for stock locations and items
* Who can edit / update a specific stock location or item depending on owner

### PR DESCRIPTION
**Pull request by eeintech**
And also updated the permissions doc with separation between part category/part and stock location/stock item.

Fixes #47

### CODE REVIEW

File: docs/releases/0.1.5.md [hunk 1]
```diff
@@ -29,6 +29,11 @@ Refer to the [build documentation](../../build/build/#overdue-builds) for more i
 
 [#1202](https://github.com/inventree/InvenTree/pull/1202) introduces the concept of an *Expiry Date* for Stock Items. For further information, refer to the [expiry documentation](../../stock/expiry).
 
+### Stock Ownership
+
+[#1155](https://github.com/inventree/InvenTree/pull/1155) adds ownership to stock locations and items. Ownership allows to control which user group or individual user can edit and manage specific stock locations and items.
+For further information, refer to the [expiry documentation](../../stock/owner).
```
Comment (SchrodingersGat) [inline docs/releases/0.1.5.md:35]: `expiry documentation` should read `ownership documentation`

File: docs/releases/0.1.5.md [hunk 2]
```diff
@@ -29,6 +29,11 @@ Refer to the [build documentation](../../build/build/#overdue-builds) for more i
 
 [#1202](https://github.com/inventree/InvenTree/pull/1202) introduces the concept of an *Expiry Date* for Stock Items. For further information, refer to the [expiry documentation](../../stock/expiry).
 
+### Stock Ownership
+
+[#1155](https://github.com/inventree/InvenTree/pull/1155) adds ownership to stock locations and items. Ownership allows to control which user group or individual user can edit and manage specific stock locations and items.
```
Comment (SchrodingersGat) [inline docs/releases/0.1.5.md:34]: `Ownership allows to control which user group .. `  would perhaps be better as `Ownership allows control over which user group ...`

File: docs/stock/expiry.md [hunk 3]
```diff
@@ -13,7 +13,7 @@ The stock expiry feature is disabled by default, and must be enabled via the set
 {% endwith %}
 
 !!! info "Non Expiring Stock"
-    If a Stock Item is not expected to expire, 
+    If a Stock Item is not expected to expire.
```
Comment (SchrodingersGat) [inline docs/stock/expiry.md:16]: Ah, this was my omission. Could you please fix this to read:

`If a Stock Item is not expected to expire, leave the expiry date field blank, or zero`

**Review by SchrodingersGat — COMMENTED** [review]
Great work overall. A couple of small changes!

**Comment by eeintech** [comment]
@SchrodingersGat Added your changes in!

**Comment by SchrodingersGat** [comment]
Great, nice work once again :)

**Comment by eeintech** [comment]
No problem, 0.1.5 release on the way? :wink:

**Comment by SchrodingersGat** [comment]
https://github.com/inventree/InvenTree/releases/tag/0.1.5

**Pull request merged**